app/src/dev/java/cm/aptoide/pt/app/view/AppViewPresenter.java

```diff
@@ -841,16 +810,15 @@ private void continueLoggedInRecommendsDialogClick() {
         .filter(lifecycleEvent -> lifecycleEvent == View.LifecycleEvent.CREATE)
         .flatMap(created -> view.continueLoggedInRecommendsDialogClick()
             .observeOn(Schedulers.io())
-            .flatMap(
-                __ -> appViewManager.recordABTestAction(ABTestManager.ExperimentType.SHARE_DIALOG))
             .flatMapSingle(__ -> appViewManager.loadAppViewViewModel())
             .observeOn(viewScheduler)
             .flatMapCompletable(app -> appViewManager.shareOnTimeline(app.getPackageName(),
                 app.getStore()
-                    .getId(), "install"))
-            .doOnNext(app -> appViewAnalytics.sendTimelineLoggedInInstallRecommendContinueEvents(
-                app.getPackageName()))
-            .doOnNext(__ -> view.showRecommendsThanksMessage())
+                    .getId(), "install")
+                .doOnCompleted(
+                    () -> appViewAnalytics.sendTimelineLoggedInInstallRecommendContinueEvents(
```

Comment:
Same as above, since there is no continue option anymore, this should be renamed.